HISTORY.md

```diff
@@ -2,6 +2,7 @@
 ## Unreleased
 ### Bug Fixes
 * Fix data corruption casued by output of intra-L0 compaction on ingested file not being placed in correct order in L0.
+* Fix a data race between Version::GetColumnFamilyMetaData() and Compaction::MarkFilesBeingCompacted() for access to being_compacted (#6056).
```

Comment:
Explain the perf regression caused by the fix.

Reply:
Done.